client/DESIGN.md

```diff
@@ -0,0 +1,111 @@
+# 🎨 UI Design Recommendation for AET Web Apps  
+
+## 🎯 Why is this Important?  
+
+- 🧭 **Easier Navigation** – Users can quickly find what they need without confusion.  
+- 🔍 **Instant Recognition** – A consistent design helps users immediately recognize they are on an AET platform.  
+- ⚡ **Improved Efficiency** – Familiar layouts and interactions reduce cognitive load, making tasks faster and more intuitive.  
+- 🎯 **Professional & Trustworthy** – A polished, cohesive UI reinforces credibility and trust in our applications.  
+- 📱 **Seamless Multi-App Experience** – Users moving between AET apps experience a unified interface, reducing friction.  
+
+By following these guidelines, we create a **seamless and enjoyable experience** for all users. 🚀  
+
+## 🏠 Landing Page  
+- **Features & FAQs** should be prominently displayed to help new users understand what the app is used for 
+- The **Login button** should be highly visible for quick access.  
+- **Most features** should be accessible **before requiring login** (avoid an early login wall).  
+
+<details>
+<summary>Example</summary>
+
+![Landing Page](design_screenshots/landing_page.png)
+</details>
+
+## Header Design  
+- Should include (in this order):
+    - **App Name**
```

Comment:
As far as i know @krusche would also like to have the version number in the header next to the app name.